packages/storybook/src/schematics/init/schema.json

```diff
@@ -3,5 +3,12 @@
   "id": "Nx Storybook Project Schematics Schema",
   "title": "Add Storybook Configuration to the workspace",
   "type": "object",
-  "properties": {}
+  "properties": {
+    "uiFramework": {
+      "type": "string",
+      "description": "Storybook UI Framework to use",
+      "enum": ["@storybook/angular", "@storybook/react", "@storybook/web"],
```

Comment:
Awesome, can I choose all?

Is `@storybook/web` "none"?

Reply:
> Awesome, can I choose all?

Good question 🤔. Rather than that, you'd probably want to pass in multiple options, like set it up for angular & react. But is that a realistic use case? Usually storybook init would be invoked anyway by the corresponding `@nrwl/angular` or `@nrwl/react` anyway. And if not, well yes then you'd have to invoke it multiple times I guess.

> Is @storybook/web "none"?

What do you mean by that?

Reply:
So for `@storybook/web` I honestly don't know how that's being used atm. I basically just took the `uiFramework` prop that the `@nrwl/storybook:configuration` already uses: https://github.com/nrwl/nx/blob/fd47229764d343a66c5d4eafc1e53147de2674b5/packages/storybook/src/schematics/configuration/schema.json#L17